### DISCUSSION TITLE
Sequential jobs?

### DISCUSSION THREAD

**Discussion by wbodger** [olahallengren/sql-server-maintenance-solution#499] [Questions & Answers]
Is there anything in the Maintenance Solution that allows you to schedule Index Optimization and then after that is finished run Integrity Checks and then after that is finished run Full Backup or do you all just create an encompassing job with those three steps?

**Comment by rishtham** [comment]
It can be done only through the scheduled job time .
you have to place each of them so that it won't collide .

---
